utils/Finder/SetRectorClassesResolver.php

```diff
@@ -21,6 +24,15 @@ final class SetRectorClassesResolver
      */
     public static function resolve(string $setFile): array
     {
+        // special level cases
+        if (realpath($setFile) === realpath(SetList::DEAD_CODE)) {
+            return DeadCodeLevel::RULES;
+        }
```

Comment:
I wonder what the difference is between `DeadCodeLevel::RULES` and `SetList::DEAD_CODE`. when use which?